pipeline/lib/pipeline/inspector.rb

```diff
@@ -10,52 +7,13 @@ def initialize(params = {})
       @params = params
       @params[:region] = ENV['AWS_REGION']
       @params[:region] ||= 'us-east-1'
-      @cloudformation = Aws::CloudFormation::Client
-                        .new(region: @params[:region])
-      @inspector = Aws::Inspector::Client.new(region: @params[:region])
 
-      setup_inspector
+      clone_inspector
       run_inspector
       cleanup_inspector
     end
 
-    def setup_inspector
```

Comment:
it functionality is all there, the redundant inspector role being assumed has all been pulled, in favor of using proper jenkins instance profile policy